export/serializers/csv_serializer.py

```diff
@@ -21,14 +21,21 @@
 import codecs
 import csv
 import re
-import StringIO
+try:
+    import StringIO
+except ImportError:
+    import io as StringIO
 
 from itertools import groupby
 from operator import itemgetter
 
 from django.core.serializers.python import Serializer as PythonSerializer
 from django.core.serializers.python import Deserializer as PythonDeserializer
-from django.utils.encoding import smart_unicode
+from django.utils import six
+try:
+    from django.utils.encoding import smart_unicode as smart_text
+except ImportError:
+    from django.utils.encoding import smart_text
```

Comment:
Again, just a matter of style: Try the python3 import first, fallback to python2.7